frontend/src/e2e-playwright/specs/4_finance/invoices.spec.ts

```diff
@@ -0,0 +1,155 @@
+      headOfChildId: fixtures.enduserGuardianFixture.id,
+      startDate: fixtures.enduserChildFixtureKaarina.dateOfBirth,
+      endDate: '2099-01-01'
+    }
+  ])
+
+  const feeDecision = feeDecisionsFixture(
+    'SENT',
+    fixtures.enduserGuardianFixture,
+    fixtures.enduserChildFixtureKaarina,
+    fixtures.daycareFixture.id
+  )
+  await insertFeeDecisionFixtures([feeDecision])
+  await insertDaycarePlacementFixtures([
+    createDaycarePlacementFixture(
+      uuidv4(),
+      fixtures.enduserChildFixtureKaarina.id,
+      fixtures.daycareFixture.id,
+      feeDecision.validDuring.start.formatIso(),
+      feeDecision.validDuring.end?.formatIso()
+    )
+  ])
+
+  page = await (await newBrowserContext({ acceptDownloads: true })).newPage()
+
+  await insertEmployeeFixture({
+    id: config.financeAdminAad,
+    externalId: `espoo-ad:${config.financeAdminAad}`,
+    email: 'lasse.laskuttaja@espoo.fi',
+    firstName: 'Lasse',
+    lastName: 'Laskuttaja',
+    roles: ['FINANCE_ADMIN']
+  })
+  await employeeLogin(page, 'FINANCE_ADMIN')
+
+  await page.goto(config.employeeUrl)
+  const nav = new EmployeeNav(page)
+  await nav.openTab('finance')
+  invoicesPage = await new FinancePage(page).selectInvoicesTab()
+})
+afterEach(async () => {
+  await page.close()
+})
+
+describe('Invoices', () => {
+  test('List of invoice drafts is empty intially and after creating new drafts the list has one invoice', async () => {
+    await invoicesPage.assertInvoiceCount(0)
+    await invoicesPage.createInvoiceDrafts()
+    await invoicesPage.assertInvoiceCount(1)
+  })
+
+  test('Navigate to and from invoice page', async () => {
+    await invoicesPage.createInvoiceDrafts()
+    await invoicesPage.openFirstInvoice()
+    await invoicesPage.assertInvoiceHeadOfFamily(
+      `${fixtures.enduserGuardianFixture.firstName} ${fixtures.enduserGuardianFixture.lastName}`
+    )
+    await invoicesPage.navigateBackToInvoices()
+  })
+
+  test('Add a new invoice row, modify its amount and price and persist the changes', async () => {
+    await invoicesPage.createInvoiceDrafts()
+    await invoicesPage.openFirstInvoice()
+    await invoicesPage.assertInvoiceRowCount(1)
+    await invoicesPage.addNewInvoiceRow('12345', 10, 100)
+    await invoicesPage.navigateBackToInvoices()
+    await invoicesPage.assertInvoiceTotal(289 + 10 * 100)
+  })
+
+  test('Delete an invoice row', async () => {
+    await invoicesPage.createInvoiceDrafts()
+    await invoicesPage.openFirstInvoice()
+    await invoicesPage.assertInvoiceRowCount(1)
+    await invoicesPage.deleteInvoiceRow(0)
+    await invoicesPage.assertInvoiceRowCount(0)
+  })
+
+  test('Invoices are toggled and sent', async () => {
+    await invoicesPage.createInvoiceDrafts()
+    await invoicesPage.toggleAllInvoices(true)
+    await invoicesPage.sendInvoices()
+    await invoicesPage.assertInvoiceCount(0)
+    await invoicesPage.showSentInvoices()
+    await invoicesPage.assertInvoiceCount(1)
+  })
```

Comment:
Would it be too much to ask for more than one invoice here to ensure the toggle _all_ actually works?